src/python/pants/backend/docker/subsystems/docker_options.py

```diff
@@ -56,6 +58,15 @@ class DockerOptions(Subsystem):
 
             A configured registry is marked as default either by setting `default = true`
             or with an alias of `"default"`.
+
+            A `docker_image` may be pushed to a subset of registries using the per registry
+            `skip_push` option rather then the all or nothing toggle of the field option `skip_push`
+            on the `docker_image` target.
+
+            Any image tags that should only be added for specific registries may be provided as the
+            `extra_image_tags` option. The tags may use value formatting the same as for the
```

Comment:
Ditto that this should probably mention the subsystem and the registry

Reply:
You mean, like the full "paths" for the options referred to? And if so, I'm not sure what that would look like here..